config/src/main/java/org/axonframework/config/Configurer.java

```diff
@@ -530,6 +531,15 @@ default Configurer registerEventHandler(Function<Configuration, Object> eventHan
         return this;
     }
 
+    default <MC extends ModuleConfiguration> MC getModuleConfiguration(Class<MC> clazz) {
```

Comment:
Missing some JavaDoc here! I might help you out here though.

Reply:
what would you recommend to write here?